src/engraving/libmscore/engravingitem.h

```diff
@@ -563,6 +563,9 @@ class EngravingItem : public EngravingObject
     std::vector<Spanner*>& startingSpanners() { return _startingSpanners; }
     std::vector<Spanner*>& endingSpanners() { return _endingSpanners; }
 
+    bool allowRemoveWhenRemovingStaves(staff_idx_t startStaff, staff_idx_t endStaff = 0) const;
```

Comment:
These functions do not feel as if they belong to this class. They are more like global functions somewhere in utils